src/providers/VernaProvider.tsx

```diff
@@ -119,9 +127,17 @@ function VernaProvider({
   locale,
   transformErrors,
   translations,
+  intl,
 }: PropsWithChildren<VernaProviderProps>) {
   // Both fullSchema & fullUiSchema are not used by the lib itself but may be
   // used for further implementation
+  const vernaIntl = createIntl(
+    {
+      ...intl,
+      messages: { ...(intl.locale === 'fr' ? fr : {}), ...intl.messages },
+    },
+    createIntlCache(),
+  );
   const [fullSchema, setFullSchema] = useState<VernaJSONSchemaType>(defaultSchema);
```

Comment:
We can do something more universal

```suggestion
  const i18n = {
    fr: fr,
    en: {},
  }

  const vernaIntl = createIntl(
    {
      ...intl,
      messages: { ...(i18n[intl.locale]), ...intl.messages },
    },
    createIntlCache(),
  );
  const [fullSchema, setFullSchema] = useState<VernaJSONSchemaType>(defaultSchema);
```